File: Exercicios/Exercicio005.cpp
// Ler um número, informar seu dobro, tripo e raiz quadrada
#include <iostream>
#include <math.h>
using namespace std;
int main(){
    int num;
    cout << "Digite um numero: ";
    cin >> num;
    cout << "-----------------------\n";
    cout << "Dobro de " << num << ": " << num*2 << "\n";
    cout << "Triplo de " << num << ": " << num*3 << "\n";
    cout << "Raiz quadrade de " << num << ": " << sqrt(num) << "\n";
    cout << "-----------------------";
    return 0;
}

File: Python/Exercicio005.py
#Ler um número inteiro e informar seu antecessor e sucessor.
import colorama
colorama.init()
num = int(input('Digite um número inteiro: '))
print(f'\033[31m{num-1}\033[m É o antecessor de \033[31;1m{num}\033[m. E seu sucessor é \033[31m{num+1}\033[m')
